mta-sts.md

```diff
@@ -617,6 +600,33 @@ A suggested workflow to implement such an opt out is as follows:
    allowing overlapping policy caches--safely remove the TXT record and HTTPS
    endpoint.
 
+## Preserving MX Candidate Traversal
+
+Implementors of send-time MTA-STS validation in mail transfer agents should take
+note of the risks of modifying the logic of traversing MX candidate lists.
+Because an MTA-STS policy can be used to prefilter invalid MX candidates from
+the MX candidate list, it is tempting to implement a "two-pass" model, where MX
+candidates are first filtered for possible validity according to the MTA-STS
+policy, and then the remaining candidates attempted in order as without an
+MTA-STS policy.  This may lead to incorrect implementations, such a message
+loops; implementors are instead recommended to traverse the MX candidate list as
+usual, and treat invalid candidates as though they were unreachable (i.e., as
+though there were some transient error when trying to deliver to that
+candidate).
+
+One consequence of maintaining MX candidate traversal order is that `report`-mode
+policies may show failures even if a message would, in `enforce`-mode,
+successfully be delivered. Consider the case of a domain with a higher-priority
+MX which is MTA-STS invalid and a lower-priority MX which is valid. In `report`
+mode, mail will be delivered to the higher-priority (invalid) host, but generate
+failure reports; in `enforce` mode, mail will successfully be delivered to the
+lower-priority (valid) host.
+
+Similarly, if a higher-priority MX is MTA-STS valid and a lower-priority MX is
```

Comment:
This next paragraph is correct. Reporting of problems for secondary MX host will only happen to the extent they're used. This will happen from time to time if the primary temp-fails some recipients, then most MTAs will retry at a secondary.
Many domains avoid the distinction between primary and backup MX hosts and use equal-weight MX records. I'd potentially recommend the same here.

Reply:
s/equal weight/equal preference/

Reply:
Done.